go/cmd/install.go

```diff
@@ -0,0 +1,196 @@
+package cmd
+
+import (
+	"archive/tar"
+	"compress/bzip2"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/flowchartsman/retry"
+	"github.com/gofrs/flock"
+	"github.com/hashicorp/go-version"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"sort"
+	"time"
+)
+
+func targetExeFilename(exeName string) string {
+	_ = os.MkdirAll(sitePath(), 0700)
+	targetFileName := filepath.Join(sitePath(), exeName)
+	if runtime.GOOS == "windows" {
+		targetFileName = targetFileName + ".exe"
+	}
+	return targetFileName
+}
+
+func InstallMicromamba() (string, error) {
+	url := fmt.Sprintf("https://micromamba.snakepit.net/api/micromamba/%s/latest", PlatformSubdir())
+	return installMicromamba(url)
+}
+
+type AnacondaPkgAttr struct {
+	Subdir       string
+	Version      string
+	Build_number int32
+	Timestamp    uint64
+	Source_url   string
+	Md5          string
+}
+
+type AnacondaPkg struct {
+	Size  uint32
+	Attrs AnacondaPkgAttr
+	Type  string
+}
+
+type AnacondaPkgAttrs []AnacondaPkgAttr
+
+func (a AnacondaPkgAttrs) Len() int { return len(a) }
+func (a AnacondaPkgAttrs) Less(i, j int) bool {
```

Comment:
nit: rename i, j -> x, y or a,b  -  makes them a little easier to distinguish for the reader

Reply:
`i`, `j` are the canonical ones for Less